apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -186,3 +211,63 @@ function CourseDirectoryMissingAlert({
     `;
   }
 }
+
+function EditActions({
+  coursePathExists,
+  courseInfoExists,
+  hasCoursePermissionView,
+  hasCoursePermissionEdit,
+  exampleCourse,
+  urlPrefix,
+  navPage,
+}: {
+  coursePathExists: boolean;
+  courseInfoExists: boolean;
+  hasCoursePermissionView: boolean;
+  hasCoursePermissionEdit: boolean;
+  exampleCourse: boolean;
+  urlPrefix: string;
+  navPage: string;
+}) {
+  if (!coursePathExists || !courseInfoExists || !hasCoursePermissionView) {
+    return;
```

Comment:
Nit, we'll soon be deprecating the usage of `null`/`undefined` in templates.

```suggestion
    return '';
```

Reply:
Updated locally in [1f9d924](https://github.com/PrairieLearn/PrairieLearn/pull/9581/commits/1f9d9242b1ab757397ce2a399f68924f58891b8e).